src/gen_idna_table.c

```diff
@@ -65,9 +67,12 @@ static void generate_set(const char *status)
            "    if (codepoint > %d)\n"
            "         return false;\n"
            "    return (set_%s[codepoint / 64] &"
-                " (uint64_t) 1 << codepoint %% 64) != 0;\n"
+           "            (uint64_t) 1 << codepoint %% 64) != 0;\n"
```

Comment:
This actually changes the generated source file to have extra spaces in the middle of a line. Is this intentional and/or should this be revised? Although, this isn't really an issue.

Reply:
Fixed.